python/pyspark/ml/classification.py

```diff
@@ -1517,20 +1517,22 @@ class OneVsRest(Estimator, OneVsRestParams, MLReadable, MLWritable):
 
     @keyword_only
     def __init__(self, featuresCol="features", labelCol="label", predictionCol="prediction",
-                 classifier=None):
+                 weightCol=None, classifier=None):
```

Comment:
Usually we append new argument at the end, as users may use non-keyword argument, we should not break users' existing code. FYI: https://docs.python.org/3/tutorial/controlflow.html#keyword-arguments